services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js

```diff
@@ -639,6 +676,12 @@ export class NOWPermitGeneration extends Component {
                     }
                     draftPermit={this.props.draftPermit}
                     draftPermitAmendment={this.props.draftPermitAmendment}
+                    storeEditingPreambleFlag={this.props.storeEditingPreambleFlag}
+                    handleSaveDraftEdit={this.handleSaveDraftEdit}
+                    handleSavePreamble={this.handleSavePreamble}
+                    handleCancelDraftEdit={this.handleCancelDraftEdit}
+                    handleCancelPreambleTextEdit={this.handleCancelPreambleTextEdit}
+                    handleChange={this.handleChange}
```

Comment:
Is this used anywhere?? Don't see it defined.

Reply:
it is not used, it was a the handler used for inserting the variable at cursor position which is not longer included.